cpp/velox/memory/VeloxMemoryPool.cc

```diff
@@ -406,16 +436,20 @@ std::shared_ptr<velox::memory::MemoryPool> AsWrappedVeloxMemoryPool(
       options);
 }
 
-std::shared_ptr<velox::memory::MemoryPool> GetDefaultWrappedVeloxMemoryPool() {
+std::shared_ptr<WrappedVeloxMemoryPool> GetDefaultWrappedVeloxMemoryPool() {
```

Comment:
`WrappedVeloxMemoryPool` is the child of `WrappedVeloxMemoryPool`, we do not need to change return value type

Reply:
Thanks, will fix it.